media_softlet/linux/common/os/osservice/mos_utilities_specific.cpp

```diff
@@ -1003,7 +1003,25 @@ MOS_STATUS MosUtilitiesSpecificNext::UserFeatureDumpDataToFile(const char *szFil
     MOS_PUF_KEYLIST   pKeyTmp;
     int32_t           j;
 
+   // For release version: writes to the file if it exists,
+// skips if it does not exist
+// For other version: always writes to the file
+
+#if(_RELEASE)
+    File = fopen(szFileName, "r");
```

Comment:
isnt that a kinda weird way to say `access()` or `stat()`?

Reply:
Hi thank you! I will use access() replace this way. May I know if the code can solve this issue which you reporter?

Reply:
all i want is not trying to open the file anymore for writing :)

so just checking with `access()` and `stat()` would be fine I guess.

Reply:
yeah, access() will be fine, I will change it.